modules/log.js

```diff
@@ -53,16 +53,23 @@ export function log() {
         callback(null, file);
       }))
       .on('finish', () => {
-        let t = 1;
+        let hasErrors = false;
+      
         for (let e in errors) {
-          let r = path.relative('_input', e);
-          console.log(`${t}. Errors in file "${r}":`);
-          for (let n of errors[e]) {
-            console.log(` > ${n}`);
+          if (errors[e].length > 0) {
+            let r = path.relative('_input', e);
+            console.log(`Errors in file "${r}":`);
+            for (let n of errors[e]) {
+              console.log(` > ${n}`);
+            }
+            console.log('--------------------------------------------------');
+            hasErrors = true;
           }
-          console.log('--------------------------------------------------');
-          t++;
         }
-      })      
+      
+        if (!hasErrors) {
+          console.log("No errors found.");
```

Comment:
Please use the following message using template literals.
```
          console.log(`--------------------------------------------------
No errors found.
--------------------------------------------------`);
```

![Screenshot 2024-11-18 at 3 23 43 PM](https://github.com/user-attachments/assets/49d77530-aa99-4d33-87a6-58f343cf8373)